### ISSUE TITLE
Name resolver changes user input

### ISSUE THREAD

**Issue by karafecho** [RobokopU24/Issues#140]
In the example below, the user (me) asked for "isopropanol" but ROBOKOP's top selection choice was for "isopropyl alcohol". While these are equivalent, a user might find this confusing. I'm not sure how significant this issue is, however. It might be best to wait for additional user feedback before attempting a fix.

![image](https://user-images.githubusercontent.com/26254388/230645081-e0e65be3-a54d-4cb4-933d-208c359e9e11.png)

**Comment by cbizon** [comment]
I agree that this is potentially confusing.   I think that we need to have a whole re-think about the entity chooser and this would be part of that.
